cabal-testsuite/PackageTests/Check/ConfiguredPackage/Fields/ImpossibleVersionRangeLib/cabal.out

```diff
@@ -1,4 +1,11 @@
 # cabal check
 The package will not build sanely due to these errors:
 Error: [impossible-dep] The package has an impossible version range for a dependency on an internal library: pkg:internal >1.0 && <2.0. This version range does not include the current package, and must be removed as the current package's library will always be used.
+These warnings may cause trouble when distributing the package:
+Warning: [tz-upper-bounds] On library, these packages have upper bounds with trailing zeros:
+  - pkg
+Please avoid trailing zeros for upper bounds.
+Warning: [gt-lower-bounds] On library, these packages have greater than (>) lower bounds:
+  - pkg
+Please use greater than or equals (>=) for lower bounds.
```

Comment:
This has to change Phil, as I believe I have mentioned in the past review.

For golden tests, we for sure prefer things *atomic*, one test for one property. Modify `pkg.cabal` accordingly, I have checked them and there should be no problem in doing it.

This has to be done for the following tests too

- `cabal-testsuite/PackageTests/Check/NonConfCheck/PackageVersionsInternal/cabal.out`
- `cabal-testsuite/PackageTests/Check/NonConfCheck/PackageVersionsInternalSimple/cabal.out`
- `cabal-testsuite/PackageTests/Check/NonConfCheck/PackageVersionsLibInt/cabal.out`
- `cabal-testsuite/PackageTests/Check/NonConfCheck/PackageVersionsStraddle/cabal.out`
- `cabal-testsuite/PackageTests/Check/NonConfCheck/SetupBounds/cabal.out`

It should be extremely simple (modify `pkg.cabal` and then rerun the testsuite with `--accept`, if you need help I can do it!

Reply:
@ffaf1, are you saying that packages in `cabal-testsuite/PackageTests/Check` that are now triggering the new check should be edited (have their version bound corrected) so that they only trigger the check they were set up for (and their `.out` is unchanged by this pull request)?

Reply:
Yes, exactly.

Reply:
I've corrected the `.cabal` files so that they don't trigger the new version range checks.